mimic.py

```diff
@@ -50,11 +50,31 @@
 def mimic_dict(filename):
     """Returns mimic dict mapping each word to list of words which follow it."""
     # +++your code here+++
+    mimic_dict = {}
+    file = open(filename, 'r')
+    sentence = file.read()
+    file.close()
+    words = sentence.split()
+    seperate = ''
+    for word in words:
+        if not seperate in mimic_dict:
+            mimic_dict[seperate] = [word]
+        else:
+            mimic_dict[seperate].append(word)
+        seperate = word
+    return mimic_dict
+
     raise NotImplementedError("Get to Work!")
```

Comment:
Note that these 'raise ...' statements are meant to be removed from final code submissions.  They are just placeholder reminders that a function needs to be implemented by the student